official/SpecificaTecnica/res/sections/activitiesDiagrams.tex

```diff
@@ -0,0 +1,353 @@
+\item invitare un utente;
+\item rimuovere un utente;
+\item inserire un utente.
+\end{itemize}
+\paragraph{Invito utente} \mbox{} \\
+\begin{figure}[H]
+\begin{center}
+\includegraphics[height=12cm]{res/sections/backend/activities/invitoUtente.png}
+\caption{Invito utente}
+\end{center}
+\end{figure}
+Un utente autenticato, owner o admin, può invitare un nuovo utente nella propria company inserendo un indirizzo email ed inviando la richiesta tramite un pulsante presente nella pagina.
+\paragraph{Inserimento utente} \mbox{} \\
+\begin{figure}[H]
+\begin{center}
+\includegraphics[height=12cm]{res/sections/backend/activities/inserimentoUtente.png}
+\end{center}
+\end{figure}
+Un utente autenticato, owner o admin, può inserire manualmente un nuovo utente nella company senza prima invitarlo. Per farlo deve fornire indirizzo email e password del nuovo utente.
+\subsubsection{Operazioni sui database}
+\begin{figure}[H]
+\begin{center}
+\includegraphics[height=12cm]{res/sections/backend/activities/gestioneDatabase.png}
+\caption{Operazioni sui database della company}
+\end{center}
+\end{figure}
+Un utente autenticato può visualizzare i database della company, e le collection di quel database, ai quali ha accesso. Gli admin e gli owner, invece, possono gestire i database registrati presso la company:
+\begin{itemize}
+\item modificando i dati di accesso e aggiungendo restrizioni sulle collections visibili ai member;
+\item aggiungendo un database;
+\item rimuovendo un database.
+\end{itemize} 
+\paragraph{Modifica database} \mbox{} \\
+\begin{figure}[H]
+\begin{center}
+\includegraphics[height=12cm]{res/sections/backend/activities/modificaDatabase.png}
+\end{center}
+\end{figure}
+Un admin o un owner può modificare un database della propria company. In particolare deve fornire:
+\begin{itemize}
+\item nome;
+\item host;
+\item porta di accesso;
+\end{itemize}
+Inoltre può aggiungere, rimuovere o modificare restrizioni sulle collections di quel database.
+\subparagraph{Aggiunta collection} \mbox{} \\
+\begin{figure}[H]
+\begin{center}
+\includegraphics[height=12cm]{res/sections/backend/activities/aggiuntaDatiCollection.png}
+\end{center}
+\end{figure}
+Un admin o un owner può decidere se rendere una nuova collection visibile ai members della company completando il form di aggiunta di una collection.
+\subparagraph{Gestione collection} \mbox{} \\
+\begin{figure}[H]
+\begin{center}
+\includegraphics[height=12cm]{res/sections/backend/activities/gestioneCollectionAdmin.png}
+\end{center}
+\end{figure}
+Un admin o un owner può modificare le regole di accesso e rimuovere le collection esistenti.
+\subparagraph{Modifica collection} \mbox{} \\
+\begin{figure}[H]
+\begin{center}
+\includegraphics[height=12cm]{res/sections/backend/activities/modificaDatiCollection.png}
+\end{center}
+\end{figure}
+Un admin o un owner può decidere se rendere una collection già esistente visibile ai members della company completando il form di modifica di una collection.
```

Comment:
- [ ] @mdipirro admin è scritto minuscolo. Non va maiuscolo?
- [ ] @mdipirro owner è scritto minuscolo. Non va maiuscolo?
- [ ] @mdipirro collection è scritto minuscolo. Non va maiuscolo? (x2)
- [ ] @mdipirro members è scritto minuscolo. Non va maiuscolo?
- [ ] @mdipirro company è scritto minuscolo. Non va maiuscolo?

- [ ] @mdipirro form è una parola inglese. Non va in corsivo?